src/components/AddBook.js

```diff
@@ -0,0 +1,19 @@
+import './addBook.css';
+
+export default function AddBook() {
+  return (
+    <div className="container">
+      <div className="line" />
+      <div className="footer-start">
+        <span className="book-title">
+          ADD NEW BOOK
+        </span>
+        <div className="input-fields">
+          <input className="book-input" placeholder="Book" />
+          <input className="author-input" placeholder="Author" />
+          <button className="add-btn" type="button">Add Book</button>
+        </div>
```

Comment:
- [ ] Based on the requirements of this project, this component should render a form with input for the title and author of the book. Kindly replace the div with a form.